packages/models-library/src/models_library/clusters.py

```diff
@@ -161,16 +173,29 @@ class Config(BaseCluster.Config):
             ]
         }
 
-    @validator("access_rights", always=True, pre=True)
+    @root_validator(pre=True)
     @classmethod
-    def check_owner_has_access_rights(cls, v, values):
+    def check_owner_has_access_rights(cls, values):
+        if values["id"] == "default":
+            return values
         owner_gid = values["owner"]
         # check owner is in the access rights, if not add it
-        if owner_gid not in v:
-            v[owner_gid] = CLUSTER_ADMIN_RIGHTS
+        access_rights = values.get("access_rights", values.get("accessRights"))
+        if owner_gid not in access_rights:
+            access_rights[owner_gid] = CLUSTER_ADMIN_RIGHTS
         # check owner has full access
-        if v[owner_gid] != CLUSTER_ADMIN_RIGHTS:
+        if access_rights[owner_gid] != CLUSTER_ADMIN_RIGHTS:
             raise ValueError(
-                f"the cluster owner access rights are incorrectly set: {v[owner_gid]}"
+                f"the cluster owner access rights are incorrectly set: {access_rights[owner_gid]}"
             )
-        return v
+        return values
+
+
+class DefaultCluster(Cluster):
```

Comment:
Should ``DefaultCluster`` be a singleton **instance**  instead of a class??
```python
default_cluster: Final[Cluster] = Cluster(id="default", name="Default cluster", ...)
```

Reply:
I would not bother with a singleton if possible since nobody likes them.
Having `Final[Cluster]` looks nice. 

Could we maybe have a factory method here? Something like:
```python
def create_default_cluster() -> Final[Cluster]:
    return Cluster(id="default", name="Default cluster", ...)
```

Reply:
I think what bothers PC is that the `DefaultCluster` can be initialised with different values that the defaults.

Reply:
ok so the thing is that ```DefaultCluster``` is the director-v2 default cluster, a.k.a. the osparc platform default cluster as defined by the ENV variables (please see this PR information).
The only constant parts it has are its name and id. The rest is defined at runtime (start of application). I will create a constant in the app then.

Reply:
Maybe you could rename it to `DeploymentCluster` or `PlatformCluster` or even `OsparcCluster` to avoid the "default" keyword.

Reply:
moved into the app. but I kept default as this is the name used all over the place.